azure-pipelines.yml

```diff
@@ -33,6 +36,10 @@ jobs:
         CONDA_ENV: cienv
         RUN_FLAKE8: yes
         RUN_CLANG_FORMAT: yes
+      py311:
+        PYTHON: '3.11'
+        CONDA_ENV: cienv
+        RUN_FLAKE8: yes
```

Comment:
This doesn't necessarily need to run `flake8` as another job does this but it is relatively harmless.